ambry-frontend/src/main/java/com/github/ambry/frontend/AmbrySecurityService.java

```diff
@@ -396,14 +397,22 @@ private void setBlobPropertiesHeaders(BlobProperties blobProperties, RestRespons
     if (blobProperties.getContentType() != null) {
       restResponseChannel.setHeader(RestUtils.Headers.AMBRY_CONTENT_TYPE, blobProperties.getContentType());
     }
-    if (blobProperties.getContentEncoding() != null) {
-      restResponseChannel.setHeader(Headers.CONTENT_ENCODING, blobProperties.getContentEncoding());
-    }
     if (blobProperties.getOwnerId() != null) {
       restResponseChannel.setHeader(RestUtils.Headers.OWNER_ID, blobProperties.getOwnerId());
     }
   }
 
+  /**
+   * Set the specific required headers for get blob info in the response.
+   * @param blobProperties the {@link BlobProperties} that need to be set in the headers.
+   * @param restResponseChannel the {@link RestResponseChannel} that is used for sending the response.
+   */
+  private void setBlobInfoHeaders(BlobProperties blobProperties, RestResponseChannel restResponseChannel) {
+    if (blobProperties.getContentType() != null) {
```

Comment:
This should be contentencoding, i think.

Reply:
My bad, fixed it.